csrc/host_ir/lower.cpp

```diff
@@ -325,4 +334,100 @@ bool isLowerableToCommunication(Expr* expr) {
   }
 }
 
+std::unique_ptr<hir::HostIrContainer> HostIrLower::lower(
```

Comment:
Can you confirm it's a straight copy, so I can skip reviewing the content?

Reply:
it is a straight copy